versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/RocksDBStore.java

```diff
@@ -106,8 +105,14 @@ public void start() {
         final List<ColumnFamilyHandle> columnFamilyHandles = new ArrayList<>();
         transactionDB = OptimisticTransactionDB.open(dbOptions, dbPath, columnFamilies, columnFamilyHandles);
         rocksDB = transactionDB.getBaseDB();
-        valueTypeToColumnFamily = IntStream.rangeClosed(1, ValueType.values().size()).boxed().collect(
-          ImmutableMap.toImmutableMap(k -> ValueType.values().get(k - 1), columnFamilyHandles::get));
+        final ImmutableMap.Builder<ValueType<?>, ColumnFamilyHandle> builder = new ImmutableMap.Builder<>();
+        for (ColumnFamilyHandle handle : columnFamilyHandles) {
+          final String valueTypeName = new String(handle.getName(), UTF_8);
+          if (!valueTypeName.equals(new String(RocksDB.DEFAULT_COLUMN_FAMILY, UTF_8))) {
+            builder.put(ValueType.values().stream().filter(v -> v.getTableName("").equals(valueTypeName)).findFirst().get(), handle);
```

Comment:
It seems that you're going the long way around here to deal with the column family name - why not change the column family name to make it easier so you can leave this as:
builder.put(ValueType.byValueName(valueTypeName), handle);

and change the creation of COLUMN_FAMILIES to v.getValueName() from v.name().

Reply:
Makes sense. I tried to align with MongoDBStore on use of TableName, but this is less work.